srcpkgs/ckb-next/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'ckb-next'
 pkgname=ckb-next
-version=0.3.1
+version=0.3.2
 revision=1
 build_style=cmake
+configure_args+=-DDISABLE_UPDATER=1
```

Comment:
This is wrong on multiple levels.

this is the first invocation of configure_args so using only '=' is safe to do and it should be between quotes